rest/dao/src/main/resources/io/syndesis/dao/deployment.json

```diff
@@ -860,13 +860,13 @@
           "group": "producer",
           "label": "producer",
           "required": true,
-          "type": "string",
+          "type": "file",
```

Comment:
what does this change imply ? Does trigger an upload button in the UI ?

Reply:
There is only one property for the `specification`, and it can accept either full specification (as String), or a URL to fetch the specification from. As per [design](https://github.com/syndesisio/syndesis/blob/master/ux/designs/apiconnector/apiconnector.md). This is an attempt to signal the UI to provide file browser for specification upload.